### PR TITLE
Normalised Example heading, added dollar sign to examples

### PR DESCRIPTION
**Pull request by RichardLitt**
License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/config.go [hunk 1]
```diff
@@ -41,15 +41,15 @@ repository.`,
 much like 'git config'. The configuration values are stored in a config
 file inside your IPFS repository.
 
-EXAMPLES:
+Examples:
 
 Get the value of the 'datastore.path' key:
 
-  ipfs config datastore.path
+  $ ipfs config datastore.path
```
Comment (dignifiedquire) [inline core/commands/config.go]: Are you sure this is a good idea? I have seen people copy and paste these things including the `$` and then wonder why it's not working.

Comment (rht) [inline core/commands/config.go]: That's true, would backticks clarify it better? Or they could have been accidentally copied as well.

Though `$` is used more often: https://github.com/git/git/blob/6ccc71a9d02726d59c1de818febf85a836f4e418/contrib/subtree/git-subtree.txt#L250, so it should be fine.

File: core/commands/config.go [hunk 2]
```diff
@@ -41,15 +41,15 @@ repository.`,
 much like 'git config'. The configuration values are stored in a config
 file inside your IPFS repository.
 
-EXAMPLES:
+Examples:
```
Comment (rht) [inline core/commands/config.go]: (btw, this is usually caps-all in man pages)

Comment (RichardLitt) [inline core/commands/config.go]: Cf: https://github.com/ipfs/community/pull/106

File: core/commands/config.go (hunk 1 above)
Comment (RichardLitt) [inline core/commands/config.go]: I think not using them in man pages is smart, but using them in READMEs makes sense. 

cf https://github.com/ipfs/community/pull/106

**Comment by whyrusleeping** [comment]
@RichardLitt wanna rebase this on latest? now that the CI should be showing green, i'm gonna start making sure its green before merging things (even though this PR doesnt really affect the tests)

**Comment by RichardLitt** [comment]
@whyrusleeping Rebased.

**Pull request merged**